### ISSUE TITLE
no-unknown-classes: entryPoint does not resolve @theme custom values in monorepo

### ISSUE THREAD

**Issue by beumerr** [schoero/eslint-plugin-better-tailwindcss#342]
## Description

When using `no-unknown-classes` with a custom `entryPoint` in a monorepo where the oxlint config is nested (e.g. `packages/config/oxlint/oxlint.config.mts`), the plugin does not resolve custom `@theme` values from the CSS entry point.

## Setup

**Monorepo structure:**
```
packages/config/oxlint/oxlint.config.mts   # oxlint config
apps/desktop/src/renderer/app/app.css       # CSS entry point
packages/config/tailwind/theme.css          # @theme definitions
```

**CSS entry point** (`app.css`):
```css
@import 'tailwindcss';
@import '../../../../../packages/config/tailwind/theme.css';
```

**Theme CSS** (`theme.css`):
```css
@theme {
  --duration-fast: 0.2s;
  --duration-default: 0.3s;
}
```

**oxlint config:**
```ts
'better-tailwindcss/no-unknown-classes': ['error', { entryPoint: 'apps/desktop/src/renderer/app/app.css' }],
```

## Behavior

- `duration-default`, `duration-fast` and other `@theme`-defined tokens are flagged as unknown
- Tailwind v4 decimal spacing values (`border-1.25`, `text-2.5`, `backdrop-blur-5`) are also flagged as unknown
- Using a config-relative path (`../../../apps/desktop/...`) resolves the theme correctly but then the plugin crashes with `ENOENT: tailwindcss/package.json` because it resolves the tailwindcss package relative to the entry point instead of `node_modules`

## Expected

Custom `@theme` values and valid Tailwind v4 spacing values should be recognized when `entryPoint` is set.

## Versions

- `eslint-plugin-better-tailwindcss`: 4.3.1
- `tailwindcss`: 4.2.1
- `oxlint`: 1.50.0

**Comment by emdede** [comment]
With the newest release, this is fixed. Thank you @schoero , great work!

**Comment by beumerr** [comment]
Thanks @schoero!
